research-hub-web/src/partial-styles/_variables.scss

```diff
@@ -323,13 +323,6 @@ $research-activity-palette-discover: (
   ),
 );
 
-/**
-* generate theme colors
```

Comment:
Hey Rob, should probably leave this here - instead of removing it, you can change the colour variables to use the Uni colours.. e.g.:
`/**
* generate theme colors
*/
$hub-app-primary: $dark-blue;
$hub-app-accent: $light-blue;
$hub-app-warn: $uni-warning-colour-whatever-that-is;`